packages/gatsby-plugin-gatsby-cloud/src/components/icons.js

```diff
@@ -0,0 +1,90 @@
+import React from "react"
+
+export const gatsbyIcon = (
```

Comment:
I disagree since this isn't a function and can't be called (i.e. you can't do: `<GatsbyIcon />`) since they are "pre-rendered" (not sure what the proper term for that is). If you wanted to render these in another component, the usage would look something like:

```js
<div>
  {gatsbyIcon}
</div>

```

I won't die on this hill, but lowercase makes more sense to me.

Reply:
You're right, I was fixated on the (<svg and missed that it wasn't a function 👍